runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerBarOverlay.java

```diff
@@ -44,8 +44,12 @@
 @Singleton
 class PrayerBarOverlay extends Overlay
 {
-	private static final Color BAR_FILL_COLOR = Color.cyan;
-	private static final Color BAR_BG_COLOR = Color.white;
+	// Color to use as the foreground "fill" of the floating prayer bar
+	private static final Color BAR_FILL_COLOR = new Color(0,149,151);
```

Comment:
needs formatting with space after commas. Configure your formatter with the xml in the wiki and `ctrl + alt + l` for future reference

Reply:
Also, the comments are unnecessary.

Reply:
Seems better to have them then to not

Reply:
Why? The variable name is very explicit on what it does. It's extraneous code.